doc/lua_api.txt

```diff
@@ -5478,9 +5478,9 @@ Used by `ObjectRef` methods. Part of an Entity definition.
         --   'inventory_image'.
         -- "item" is similar to "wielditem" but ignores the 'wield_image' parameter.
 
-        visual_size = {x = 1, y = 1},
-        -- `x` multiplies horizontal (X and Z) visual size.
-        -- `y` multiplies vertical (Y) visual size.
+        visual_size = {x = 1, y = 1, z = 1},
+        -- Multipliers for the visual size. If `z` is not specified, `x` will be used
+        -- to scale the entity along both horizontal axes.
```

Comment:
"... along both `x` and `z` axes" would sound less confusing to me, because I'm used to coordinate systems with `z` vertical. I'm not sure whether it would be clearer that way for everyone else too.

Reply:
Seems fine to me as it is.